r-pkg/uptasticsearch/man/parse_date_time.Rd

```diff
@@ -24,8 +24,8 @@ Given a data.table with date-time strings,
 \dontrun{
 
 # Get some data
-result <- es_search(es_host = "http://es.custdb.mycompany.com:9200"
-                 , es_index = 'cust_details')
+someDT <- es_search(es_host = "http://es.custdb.mycompany.com:9200"
```

Comment:
no I just made this change in this PR. As I was going through the various examples